tools/ui-components/src/toggle-button/types.ts

```diff
@@ -0,0 +1,13 @@
+export type ButtonVariant = 'primary' | 'danger';
+
+export type ButtonSize = 'small' | 'medium' | 'large';
+
+export interface ToggleButtonProps {
+  children: React.ReactNode;
```

Comment:
There is one missing prop `value`, used in TB

```suggestion
export interface ToggleButtonProps {
  children: React.ReactNode;
  value?: number;
```

Reply:
> I think the `value` type is `any` in the bootstrap implementation.

True, but there is no reason to go for something unsafe as `any`, while we know that the type in the codebase is `number`.

> I tried setting it anyway

because I don't know, how did you set it up. I can't give a good judgment on why the test is falling.

Reply:
I am stuck here, so asked this in the Headless UI discussion https://github.com/tailwindlabs/headlessui/discussions/2104

I am not sure how would the value be accessible in the code.

Reply:
Value in switch is used as string, while value in bootstrap doesn't type, as you mentioned before, while we use it with for number type but as boolean value, too. You can ignore this for now.

because the alternative that I can think of is to use `checked` as `boolean` in our codebase instead of `value`.

Reply:
@rajgaur98, thank you for your patience.

You can use type `string` with the `value` prop. When you are done with suggested changes (removing the `^` and add `bs` to style and size props, we can do a 2nd round of review.

Reply:
Thanks @Sboonny , I have updated the code